### ISSUE TITLE
Babel dependency is throwing errors for `3.15.2`+

### ISSUE THREAD

**Issue by Blackbaud-TrevorBurch** [blackbaud/skyux-sdk-builder#233]
The `babel` dependencies introduced in `3.15.2` have started throwing errors for consumers. This renders versions after `3.15.2` unusable at the moment.

Example error:

```
Uncaught Error: Module build failed (from ./node_modules/babel-loader/lib/index.js):
Error: [BABEL] D:\Git\skyux-spa-ems-checklists\node_modules\strip-ansi\index.js: Could not find plugin "proposal-numeric-separator". Ensure there is an entry in ./available-plugins.js for it. (While processing: "D:\\Git\\skyux-spa-ems-checklists\\node_modules\\@babel\\preset-env\\lib\\index.js")
    at getPlugin (D:\Git\skyux-spa-ems-checklists\node_modules\@babel\preset-env\lib\index.js:67:11)
    at Array.from.map.pluginName (D:\Git\skyux-spa-ems-checklists\node_modules\@babel\preset-env\lib\index.js:258:62)
    at Array.map (<anonymous>)
    at _default (D:\Git\skyux-spa-ems-checklists\node_modules\@babel\preset-env\lib\index.js:258:43)
    at D:\Git\skyux-spa-ems-checklists\node_modules\@babel\helper-plugin-utils\lib\index.js:19:12
    at D:\Git\skyux-spa-ems-checklists\node_modules\@babel\core\lib\config\full.js:199:14
    at Generator.next (<anonymous>)
    at Function.<anonymous> (D:\Git\skyux-spa-ems-checklists\node_modules\@babel\core\lib\gensync-utils\async.js:26:3)
    at Generator.next (<anonymous>)
    at step (D:\Git\skyux-spa-ems-checklists\node_modules\gensync\index.js:254:32)
    at Object../node_modules/strip-ansi/index.js (https://localhost:8000/ems-checklists/polyfills.js:16127:7)
    at __webpack_require__ (https://localhost:8000/ems-checklists/polyfills.js:727:30)
    at fn (https://localhost:8000/ems-checklists/polyfills.js:101:20)
    at Object.<anonymous> (https://localhost:8000/ems-checklists/polyfills.js:17039:17)
    at Object../node_modules/webpack-dev-server/client/index.js?https://localhost:8000 (https://localhost:8000/ems-checklists/polyfills.js:17210:30)
    at __webpack_require__ (https://localhost:8000/ems-checklists/polyfills.js:727:30)
    at fn (https://localhost:8000/ems-checklists/polyfills.js:101:20)
    at Object.0 (https://localhost:8000/ems-checklists/polyfills.js:21234:1)
    at __webpack_require__ (https://localhost:8000/ems-checklists/polyfills.js:727:30)
    at https://localhost:8000/ems-checklists/polyfills.js:794:37
```

**Comment by Blackbaud-BobbyEarl** [comment]
Others having the same issue:

- https://github.com/angular/angular-cli/issues/17262
- https://www.reddit.com/r/angular/comments/flymj5/build_failing_please_help/?utm_source=share&utm_medium=ios_app&utm_name=iossmf
- https://github.com/facebook/create-react-app/issues/8680

It's unclear if the `resolutions` fix others have suggested will work for all consumers without the "force-resolutions" package.

I'm digging in now but was really hoping for a global solution to the problem since other, unrelated to SKY UX, packages are having the issue.

**Comment by Blackbaud-BobbyEarl** [comment]
Addressed in [`3.16.2`](https://github.com/blackbaud/skyux-sdk-builder/blob/3.16.2/CHANGELOG.md#3162-2020-03-23)
